2022-01-03 10:34:16:530 - info: main : redis-connect-database = 0 : 49.50.172.145:6379
2022-01-03 10:34:16:535 - info: sub : redis-connect-database = 1 : 49.50.172.145:6379
2022-01-03 10:34:16:537 - info: connect sub redis : ["server_message"]
2022-01-03 10:34:16:539 - info: msg : redis-connect-database = 2 : 49.50.172.145:6379
2022-01-03 10:34:16:543 - info: mysql freemeta_com com db init : 49.50.172.145:3306
2022-01-03 10:34:16:543 - info: mysql freemeta_log log db init : 49.50.172.145:3306
2022-01-03 10:34:16:543 - info: mysql freemeta_msg msg db init : 49.50.172.145:3306
2022-01-03 10:34:16:557 - info: mysql freemeta_01 shard db init : 10.36.69.58:3306
2022-01-03 10:34:16:578 - info: load achievement reference table success !!
2022-01-03 10:34:16:584 - info: load character reference table success !!
2022-01-03 10:34:16:597 - info: load collection reference table success !!
2022-01-03 10:34:16:604 - info: load daily attendance reference table success !!
2022-01-03 10:34:16:655 - info: load item reference table success !!
2022-01-03 10:34:16:665 - info: load open world reference table success !!
2022-01-03 10:34:16:672 - info: load payment reference table success !!
2022-01-03 10:34:16:679 - info: load play world reference table success !!
2022-01-03 10:34:16:686 - info: load ranking reward reference table success !!
2022-01-03 10:34:16:704 - info: load reward reference table success !!
2022-01-03 10:34:16:713 - info: load reward_fishing reference table success !!
2022-01-03 10:34:16:719 - info: load system shop reference table success !!
2022-01-03 10:34:17:071 - info: 서버가 시작되었습니다. [port:30051]
2022-01-03 10:34:17:071 - info: try to start [freemeta game test mode server] : http://127.0.0.1:30051
2022-01-03 10:34:17:072 - info: packet status : packet status not recorded yet.
2022-01-03 10:36:05:139 - error: {"cmd":615,"aidx":139,"session":"24394567","collection_code":101001,"seq":1}
2022-01-03 10:36:05:139 - error: {"data":{},"cmd":616,"status":-1081}
2022-01-03 10:36:05:140 - error: -1081
2022-01-03 10:36:05:142 - error: logdb_connect error : {"errorno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","fatal":true}
